README.md

```diff
@@ -84,13 +84,27 @@ For more models, go to [**tensorflow/swift-models**](https://github.com/tensorfl
 
 ### Building and testing
 
+#### Building using swift-package-manager (swift-build)
+
 ```
 $ swift build
 ```
 ```
 $ swift test
 ```
 
+#### Building using CMake
+
+Note that this requires CMake 3.15.1 or newer.  In-tree builds are not
```

Comment:
I wonder what "in-tree build" means? I found https://unix.stackexchange.com/a/208643 but it doesn't quite explain.

Reply:
It just means that the build intermediates and artifacts are in the source tree.